vendors/pc/boards/windows/aws_demos/application_code/runtimeTimer.c

```diff
@@ -0,0 +1,70 @@
+#include "runtimeTimer.h"
+
+#include "FreeRTOS.h"
+#include "FreeRTOSIPConfig.h"
+#include "task.h"
+#include "timers.h"
+
+#define MS_PER_MINUTE (SECONDS_PER_MINUTE * MS_PER_SECOND)
+#define SECONDS_PER_MINUTE (60)
+#define MS_PER_SECOND (1000)
+
+#define RUNTIME_TASK_INTERRUPT_NUMBER (3)
```

Comment:
If this system had more interrupts, I would place this value in an `interrupts.h` file instead.